installer/MSIX/PackagingLayout.xml

```diff
@@ -16,12 +16,13 @@
         <File DestinationPath="modules\PowerRenameExt.dll" SourcePath="..\..\x64\Release\modules\PowerRenameExt.dll"/>
         <File DestinationPath="modules\shortcut_guide.dll" SourcePath="..\..\x64\Release\modules\shortcut_guide.dll"/>
         <File DestinationPath="modules\PowerRenameUWPUI.exe" SourcePath="..\..\x64\Release\PowerRenameUWPUI.exe"/>
-        <File DestinationPath="modules\ImageResizer.exe" SourcePath="..\..\x64\Release\ImageResizer.exe"/>
+        <File DestinationPath="modules\ImageResizer.exe" SourcePath="..\..\x64\Release\modules\ImageResizer.exe"/>
         <File DestinationPath="modules\ImageResizerExt.dll" SourcePath="..\..\x64\Release\modules\ImageResizerExt.dll"/>
-        <File DestinationPath="modules\GalaSoft.MvvmLight.dll" SourcePath="..\..\x64\Release\GalaSoft.MvvmLight.dll"/>
-        <File DestinationPath="modules\GalaSoft.MvvmLight.Platform.dll" SourcePath="..\..\x64\Release\GalaSoft.MvvmLight.Platform.dll"/>
-        <File DestinationPath="modules\GalaSoft.MvvmLight.Extras.dll" SourcePath="..\..\x64\Release\GalaSoft.MvvmLight.Extras.dll"/>
-        <File DestinationPath="modules\System.Windows.Interactivity.dll" SourcePath="..\..\x64\Release\System.Windows.Interactivity.dll"/>
+        <File DestinationPath="modules\GalaSoft.MvvmLight.dll" SourcePath="..\..\x64\Release\modules\GalaSoft.MvvmLight.dll"/>
+        <File DestinationPath="modules\GalaSoft.MvvmLight.Platform.dll" SourcePath="..\..\x64\Release\modules\GalaSoft.MvvmLight.Platform.dll"/>
+        <File DestinationPath="modules\GalaSoft.MvvmLight.Extras.dll" SourcePath="..\..\x64\Release\modules\GalaSoft.MvvmLight.Extras.dll"/>
```

Comment:
nit: Can we copy a entire folder instead of individual `dll` like copy `..\..\x64\Release\modules\ImageResizer\*` to `modules\ImageResizer\*`

Reply:
Since most of the dlls are for libraries by shifting it to a separate ImageResizer folder if another powertoy uses any of these libraries (for example MvvmLight) a duplicate copy of the dll will be present in the modules\NewToy folder. Doesn't it make most sense for the dlls to be present in the same directory?
Another change that would be needed if we shift to a modules\ImageRes folder is that the resources.dll files will also have to be moved, but we ideally want a single folder for each language which has the resources.dll files for each of the .NET powertoys